packages/host/tests/acceptance/interact-submode-test.gts

```diff
@@ -1349,4 +1349,22 @@ module('Acceptance | interact submode tests', function (hooks) {
         .hasText('FadhlanXXX');
     });
   });
+
+  module('workspace index card', function () {
+    test('cannot be deleted', async function (assert) {
+      await visitOperatorMode({
+        stacks: [
+          [
+            {
+              id: `${testRealmURL}index`,
+              format: 'isolated',
+            },
+          ],
+        ],
+      });
+      await click('[data-test-more-options-button]');
+      assert.dom('[data-test-boxel-menu-item-text="Delete"]').doesNotExist();
+      await click('[data-test-close-button]');
```

Comment:
Not sure if clicking the close button brings any value to this test?

Reply:
Good point, actually it does not - let me remove that one